File: src/AudioraMobile/app/src/main/java/com/audiora/app/models/AuthResponse.java
package com.audiora.app.models;

public class AuthResponse {
    private String token;
    private String name;
    private String email;
    private String role;
    private boolean isPremium;

    public String getToken()   { return token; }
    public String getName()    { return name; }
    public String getEmail()   { return email; }
    public String getRole()    { return role; }
    public boolean isPremium() { return isPremium; }
}

File: src/Audiora.Admin/Models/AuthResponse.cs
namespace Audiora.Admin.Models;

public class AuthResponse
{
    public string Token     { get; set; } = "";
    public string Name      { get; set; } = "";
    public string Email     { get; set; } = "";
    public string Role      { get; set; } = "";
    public bool   IsPremium { get; set; }
}